### DISCUSSION TITLE
Benchmarking for Antmedia price/performance for instances

### DISCUSSION THREAD

**Discussion by socalengineer** [ant-media/Ant-Media-Server#5377] [FAQs]
Hi,

Does anyone have any benchmarks for price/performance for instances?

ie: how many broadcasters can be input to a server with 16-cores? With an ABR ladder of 480p, 720p, 1080p, etc?

Thanks

**Comment by USAMAWIZARD** [comment]
Let me check this and update the results here by tomorrow

**Comment by USAMAWIZARD** [comment]
A 16-cores server with ABR of 480p, 720p, 1080p, can approximately handle around 15 to 18 Live Streams at a time.
I have tested this on c4.4xlarge  which has 16v CPUs and 30 GB of memory.
with around 15 streams CPU usage was around  75 to 80 %
15 Publishers and no viewers
